.github/workflows/tests-e2e-upgrade.yaml

```diff
@@ -81,13 +81,15 @@ jobs:
             kube-proxy: 'iptables'
             kpr: 'false'
             tunnel: 'vxlan'
+            host-fw: 'true'
```

Comment:
Could you please comment out the changes in the `tests-e2e-upgrade.yaml` file (I'd not delete them altogether because of the keep in sync note above, as it could cause confusion in the future). The reason being that https://github.com/cilium/cilium/pull/30818 was not backported to v1.14, so I'm pretty confident it would always fail.